docs/proposals/ReID-example-for-Sedna.md

```diff
@@ -0,0 +1,88 @@
+# End-to-end Example For Sedna Re-ID Feature
+
+ ## Motivation
+
```

Comment:
You can figure out the difficulty and complexity of this scenario, such as:
It will be compared n * m targets, but the old version only handle 1 * m targets at one search.